datahub-frontend/app/controllers/CentralLogoutController.java

```diff
@@ -0,0 +1,44 @@
+package controllers;
+
+import com.typesafe.config.Config;
+import org.pac4j.play.LogoutController;
+import play.mvc.Result;
+
+import javax.inject.Inject;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * Responsible for handling logout logic with oidc providers
+ */
+public class CentralLogoutController extends LogoutController {
+
+  private static final String AUTH_BASE_URL_CONFIG_PATH = "auth.baseUrl";
+  private static final String DEFAULT_BASE_URL_PATH = "/";
+  private static Boolean _isOidcEnabled = false;
+
+  @Inject
+  public CentralLogoutController(Config config) {
+
+    String _authBaseUrl = config.hasPath(AUTH_BASE_URL_CONFIG_PATH)
+            ? config.getString(AUTH_BASE_URL_CONFIG_PATH)
+            : DEFAULT_BASE_URL_PATH;
+
+    _isOidcEnabled = config.hasPath("auth.oidc.enabled") && config.getBoolean("auth.oidc.enabled");
```

Comment:
Can we reuse some static constants for these string values?

Reply:
I dont see how they can be reused at the moment.

Reply:
They should be inside the OidcConfigs.java class as public static strings